distributed/comm/tests/test_ucx_config.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+from time import sleep
+
+ucp = pytest.importorskip("ucp")
+rmm = pytest.importorskip("rmm")
+
+import dask
+from dask.utils import format_bytes
+from distributed import Client, Worker, Scheduler
+from distributed.utils_test import gen_test, loop, inc, cleanup, popen  # noqa: 401
+from distributed.utils import get_ip
+
+try:
+    HOST = get_ip()
+except Exception:
+    HOST = "127.0.0.1"
+
+
+@pytest.mark.asyncio
+async def test_ucx_config(cleanup):
+    ucx = {
+        "nvlink": True,
+        "infiniband": True,
+        "tcp-over-ucx": True,
+        "net-devices": "mlx5_0:1",
+    }
+
+    with dask.config.set(ucx=ucx):
+        async with Scheduler(protocol="ucx") as s:
+            async with Worker(s.address) as a:
+                async with Client(s.address, asynchronous=True) as c:
+
+                    # scheduler is configured with NVLINK/IB
+                    ucx_config = await c.run_on_scheduler(ucp.get_config)
+                    assert ucx_config.get("TLS") == "rc,tcp,sockcm,cuda_copy,cuda_ipc"
+                    assert ucx_config.get("SOCKADDR_TLS_PRIORITY") == "sockcm"
+                    assert ucx_config.get("NET_DEVICES") == "mlx5_0:1"
+
+                    # worker is configured with NVLINK/IB
+                    worker_ucx_config = await c.run(ucp.get_config)
+                    ucx_config = worker_ucx_config[a.contact_address]
+                    assert ucx_config.get("TLS") == "rc,tcp,sockcm,cuda_copy,cuda_ipc"
+                    assert ucx_config.get("SOCKADDR_TLS_PRIORITY") == "sockcm"
+                    assert ucx_config.get("NET_DEVICES") == "mlx5_0:1"
```

Comment:
No, this test will fail.  The tests here having problems running consecutively -- still trying to understand wha the issue is.  Currently testing on a DGX machine

Reply:
I'm not sure you'll be able to easily test this. You'd need to find what's the main network interface somehow and use that. I suggest you skip this test if it's not a DGX or allow it to xfail. This is what we do in dask-cuda as well because we don't have a way to identify the interfaces easily.